extension/chrome/elements/add_pubkey.ts

```diff
@@ -35,10 +35,16 @@ View.run(class AddPubkeyView extends View {
   public render = async () => {
     Ui.event.protect();
     for (const missingPubkeyEmail of this.missingPubkeyEmails) {
-      Xss.sanitizeAppend('select.email', `<option value="${Xss.escape(missingPubkeyEmail)}">${Xss.escape(missingPubkeyEmail)}</option>`);
+      const escapedMissingPubkeyEmail = Xss.escape(missingPubkeyEmail);
+      Xss.sanitizeAppend('select.email', `<option value="${escapedMissingPubkeyEmail}">${escapedMissingPubkeyEmail}</option>`);
     }
+    const uniqueEmails = new Set<string>();
     for (const contact of await ContactStore.search(undefined, { hasPgp: true })) {
-      Xss.sanitizeAppend('select.copy_from_email', `<option value="${Xss.escape(contact.email)}">${Xss.escape(contact.email)}</option>`);
+      uniqueEmails.add(contact.email);
+    }
+    for (const email of Array.from(uniqueEmails).sort()) {
+      const escapedEmail = Xss.escape(email);
+      Xss.sanitizeAppend('select.copy_from_email', `<option value="${escapedEmail}">${escapedEmail}</option>`);
```

Comment:
That's the first part, all right.
And where is the second part, as specified by Tom's comment?
```
Then all public keys of that recipient should be copied over for the other recipient.
```
I suggest to not copy revoked keys, `ContactStore.getEncryptionKeys` would be enough.
Also, `<textarea class="pubkey" ` should be invisible when an email from "Copy From Contact" is selected, as it isn't appropriate for multiple pubkeys (we discussed that we'll no longer use `ContactStore.get` here)

Reply:
> That's the first part, all right. And where is the second part, as specified by Tom's comment?
> 
> ```
> Then all public keys of that recipient should be copied over for the other recipient.
> ```
> 
> I suggest to not copy revoked keys, `ContactStore.getEncryptionKeys` would be enough. Also, `<textarea class="pubkey" ` should be invisible when an email from "Copy From Contact" is selected, as it isn't appropriate for multiple pubkeys (we discussed that we'll no longer use `ContactStore.get` here)

@rrrooommmaaa Please recheck now